src/style.css

```diff
@@ -0,0 +1,112 @@
+/* Fonts */
+@font-face {
+  font-family: Avenir;
+  font-weight: 300;
+  src: url(fonts/Avenir-Book.ttf);
+}
+
+@font-face {
+  font-family: Avenir;
+  src: url(fonts/Avenir-Heavy.ttf);
+  font-weight: 900;
+}
+
+/* Global Styles */
+
+body {
+  margin: 0;
+  font-weight: 300;
+}
+
+.main {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  margin: 20px 8px;
+}
+
+.search {
+  width: 100%;
+  margin-bottom: 20px;
+}
+
+/* Search Styles */
+
+.search__input {
+  box-sizing: border-box;
```

Comment:
better init box-sizing: border-box in `*` selector for all html elements

Reply:
done